portfolio/src/main/webapp/index.html

```diff
@@ -9,7 +9,26 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>Hello I am Gabriel Curtis</p>
+      <p>This is my portfolio!</p>
+      <ul>
+          <li>I don't like writing that much, but I like reading</li>
+          <li>Math enthusiast and Lover of Algorithms</li>
+          <li>I love all kinds of music, especially rock and classical</li>
+          <li>Favorite director is Christopher Nolan</li>
+          <li>2nd Place table tennis champion at George Mason University</li>
+      </ul>
+      <a src="https:/github.com/gccurtis">My github account: gccurtis</a>
+      <a src="https://www.linkedin.com/in/gabriel-curtis-8a8586190/">This is my LinkedIn account</a>
+      <br>
+      <p>My Gallery</p>
```

Comment:
Should this be a header rather than a paragraph?

Reply:
Will fix